TOC-tidb-cloud.md

```diff
@@ -90,6 +90,7 @@
   - [Scale a TiDB Cluster](/tidb-cloud/scale-tidb-cluster.md)
   - [Upgrade a TiDB Cluster](/tidb-cloud/upgrade-tidb-cluster.md)
   - [Delete a TiDB Cluster](/tidb-cloud/delete-tidb-cluster.md)
+  - [Use TiDB Cloud API (Beta)](/tidb-cloud/api-overview.md)
```

Comment:
IMO, it is kindly redundant.
What we need is put API usage in every docs explaining each operation, for example:
![image](https://user-images.githubusercontent.com/867381/185834235-c8b07bc4-5fa4-4f86-9238-bcb08926609a.png)

Reply:
Will do later. But I'm not sure if it's ok to put API usage in every doc when it is still in _beta_.